src/build.ts

```diff
@@ -51,40 +51,35 @@ async function emitFlavor(
   mergeNamesakes(exposed);
   exposed.events = webidl.events;
 
-  const result = emitWebIdl(
-    exposed,
-    options.global[0],
-    "",
-    options.compilerBehavior,
-  );
-  await fs.writeFile(
-    new URL(`${options.name}.generated.d.ts`, options.outputFolder),
-    result,
-  );
-
-  const iterators = emitWebIdl(
-    exposed,
-    options.global[0],
-    "sync",
-    options.compilerBehavior,
-  );
-  await fs.writeFile(
-    new URL(`${options.name}.iterable.generated.d.ts`, options.outputFolder),
-    iterators,
-  );
+  // Helper to emit and write multiple iterators
```

Comment:
```suggestion
  // Iterator types in separate files as the default target doesn't understand iterators (for TS 6.0-)
```

Probably should be formatted though.